TreeDB/cmd/treedb_rag_benchmark/application_fixture.go

```
@@ -0,0 +1,485 @@
+}
+
+func loadSemanticVectors() (*semanticVectorBundle, error) {
+	var bundle semanticVectorBundle
+	if err := json.Unmarshal(semanticVectorsJSON, &bundle); err != nil {
+		return nil, fmt.Errorf("semantic vectors: decode: %w", err)
+	}
+	return &bundle, nil
+}
+
+func (b *semanticVectorBundle) Digest() string {
+	raw, _ := json.Marshal(b)
```

golangci-lint (2.12.2)

[error] Error return value of `encoding/json.Marshal` is not checked: unsafe type `float32` found

(errchkjson)